pages.zh/common/adb-reverse.md

```diff
@@ -0,0 +1,25 @@
+# adb reverse
+
+> Android Debug Bridge Reverse: reverse socket connections from an Android emulator instance or connected Android devices.
+> 详见: <https://developer.android.com/studio/command-line/adb>.
+
+- 显示所有来自设备的(DEVICE —> PC)映射的端口连接列表
+
+`adb reverse —list`
+
+- 反向映射端口连接(DEVICE —> PC)
+
+`adb reverse (remote) (local)`
+
+`adb reverse tcp:7000 tcp:5000`
```

Comment:
The tldr pages have an style guide, and we expect 1 example description and 1 command example. And the translation should be just `translation` from the primary (English) pages. In this page I see you try to explain the command using another approach and 2 examples for description etc. Could you please check that you just translate the primary pages?

Reply:
nice comment.I will change it